src/Servers/IIS/AspNetCoreModuleV2/RequestHandlerLib/environmentvariablehash.h

```diff
@@ -13,6 +13,7 @@
 #define ASPNETCORE_IIS_AUTH_BASIC                   L"basic;"
 #define ASPNETCORE_IIS_AUTH_ANONYMOUS               L"anonymous;"
 #define ASPNETCORE_IIS_AUTH_NONE                    L"none"
+#define ANCM_PREFER_ENVIRONMENT_VARIABLES_ENV_STR   L"ANCM_PREFER_ENVIRONMENT_VARIABLES"
```

Comment:
Why `ANCM` instead of `ASPNETCORE_ANCM` like in `ASPNETCORE_ANCM_HTTPS_PORT`?

Reply:
🤷‍♂ @jkotalik @anurse 
Let the naming bikeshed commence!

Reply:
This environment variable is only used by ANCM. All other environment variables are eventually used by managed code, while this is exclusively used by ANCM.

Reply:
I'm fine with this name for now.

Reply:
We use ANCM for other variables I believe, right? Like [`ANCM_HTTPS_PORT`](https://github.com/dotnet/aspnetcore/search?q=ANCM_HTTPS_PORT&unscoped_q=ANCM_HTTPS_PORT).

Seems reasonable to keep this name consistent.

Reply:
> Why `ANCM` instead of `ASPNETCORE_ANCM` like in `ASPNETCORE_ANCM_HTTPS_PORT`?

Ah, and the reason for the `ASPNETCORE_` prefix in `HTTPS_PORT` is the fact that it's read out of config. This is directly read as an env var. I could go either way I suppose 🤷‍♀